src/app/public/plugin-resources/code-examples/alert/basic/alert-demo.module.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'sky-alert-demo',
+  templateUrl: './alert-demo.component.html'
+})
+export class SkyAlertDemoComponent {
```

Comment:
The name of this file is `alert-demo.module.ts`. Is that correct?

Reply:
There should be a module file and a component file. See: https://github.com/blackbaud/skyux-popovers/blob/master/src/app/public/plugin-resources/code-examples/popover/basic/popover-demo.module.ts

Reply:
There is a component and module file? I think everything is correct, let me know if I've missed something

Reply:
Oh I see! I will resolve this thanks!